resources/function_help/json/main_angle

```diff
@@ -2,7 +2,7 @@
   "name": "main_angle",
   "type": "function",
   "groups": ["GeometryGroup"],
-  "description":"Returns the main angle of a geometry (clockwise, in degrees from North), which represents the angle of the oriented minimal bounding rectangle which completely covers the geometry.",
+  "description":"Returns the angle of the long axis (clockwise, in degrees from North) of the oriented minimal bounding rectangle, which completely covers the geometry.",
```

Comment:
Looks clearer to me indeed. Thanks @roya0045 
Question: is the comma needed? The _which_ stands for _rectangle_, isn't it? Should they then be separated? That said, I'm not a native speaker/writer

Reply:
Since 'which completely covers the geometry' is additionnal information regarding the rectangle I added a comma to separate this. That rule exists in both french and english. I also felt that the phrase was too long otherwise.